.env

```diff
@@ -71,6 +71,7 @@ ODOO_CRM_URL=
 ODOO_CRM_DB=
 ODOO_CRM_USER=
 ODOO_CRM_PASSWORD=
+# SLACK_HOOK_URLS=\#infrastructure=https://example.com,\#team=https://example.org
```

Comment:
If we want it also in prod, we should add it to env/env.off (and other flavors ?)

Reply:
@alexgarel Can you expand on how the secrets should be handled for the production config? It doesn't seem like the hook-URLs should be in git

Reply:
So my proposal:

Keep the env variable for docker, as you did, but read it in Config2_docker. When we deploy we can add it if Config2_oxf.pm on each server (we can do it before deploying indeed).

When we deploy in production,